doc/source/development/building_from_source.rst

```diff
@@ -2210,6 +2210,37 @@ First refer to https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.htm
 and to https://github.com/OSGeo/gdal/blob/master/.github/workflows/android_cmake/start.sh for
 an example of a build script to cross-compile from Ubuntu.
 
+
+Typical build issues
+++++++++++++++++++++
+
+How do I get PROJ ?
```

Comment:
This may fit more naturally in `dev_environment.rst`

Reply:
I'm not sure. dev_environment has this connotation this is for people who want to modify GDAL, but here it is problems for people who just want to build GDAL for use

Reply:
Maybe a better name for that page is "Setting up a build environment" ?

Reply:
indeed. I'll change that in an extra commit

Reply:
hum actually not, the content is rather dev oriented, with Vagrant, git, etc. Hard to organize properly! And difficult to draw the line between "user just compiling GDAL to use it because not happy with existing packages" and "developer wanting to hack in GDAL"